mlflow/models/evaluation/default_evaluator.py

```diff
@@ -1159,6 +1159,8 @@ def _get_args_for_metrics(self, extra_metric, eval_df):
                         and column in self.other_output_columns.columns
                     ):
                         eval_fn_args.append(self.other_output_columns[column])
+                    elif self.evaluator_config.get(column, None) is not None:
+                        eval_fn_args.append(self.evaluator_config.get(column))
```

Comment:
I know we don't really do this too much elsewhere, but would it be possible to annotate this function with some comments explaining what each case is supposed to do?

Maybe this is more of a task for a follow-up Jira ticket, but I do think that this function is has become a little difficult to follow.